src/SecTester.Core/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -15,5 +19,28 @@ public static IServiceCollection AddSecTesterConfig(this IServiceCollection coll
       collection.Add(new ServiceDescriptor(typeof(Configuration), configuration));
       return collection;
     }
+
+    public static IServiceCollection AddSystemTimeProvider(this IServiceCollection collection, SystemTimeProvider? instance = null)
+    {
+      collection.AddSingleton(instance ?? new UtcSystemTimeProvider());
+      return collection;
+    }
+
+    public static IServiceCollection AddDefaultLogging(this IServiceCollection collection, LogLevel logLevel = LogLevel.Error)
+    {
+      
+      collection.AddLogging((builder) =>
+      {
+        builder.AddConsoleFormatter<ColoredConsoleFormatter, DefaultConsoleFormatterOptions>();
+        builder.SetMinimumLevel(logLevel);
+        builder.AddConsole(options =>
```

Comment:
Please use a chain of calls where it is possible

```suggestion
        builder.SetMinimumLevel(logLevel).AddConsole(options =>
```